recipes/cobble/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "cobble" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cobble-{{ version }}.tar.gz
+  sha256: 52e7f886adbe42ccc3675dd627c87f6db8f22f96b6e0754a9e642ac7dd94fa42
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - cobble
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/mwilliamson/python-cobble
+  summary: Create data objects
+  license: JSON
```

Comment:
It appears that the license should be a 2-Clause BSD: `BSD-2-Clause`. Where did you get the text of `license.txt` from?